### PR TITLE
feat(gatsby-remark-prismjs): load languages when referred to by an alias

### PR DESCRIPTION
**Pull request by sidharthachatterjee**
Currently, referring to a language by an alias (defined in https://github.com/PrismJS/prism/tree/master/components) doesn't work unless the language is already loaded. 

This PR adds first class support for aliases so that languages are loaded when they are referred to by their aliases as well. 

Fixes https://github.com/gatsbyjs/gatsby/issues/13119

### CODE REVIEW

File: packages/gatsby-remark-prismjs/src/__tests__/load-prism-language.js [hunk 1]
```diff
@@ -41,4 +41,32 @@ describe(`load prism language`, () => {
     expect(Prism.languages).toHaveProperty(requiredLanguage)
     expect(languagesAfterLoaded.length).toBe(languagesBeforeLoaded.length + 2)
   })
+
+  it(`loads languages when referred by their aliases (set as a String in components)`, () => {
```
Comment (sidharthachatterjee) [inline packages/gatsby-remark-prismjs/src/__tests__/load-prism-language.js]: Sorry about these test names 🤦‍♂️ 

If you think of better ones, please recommend!

**Comment by sidharthachatterjee** [comment]
So tests are failing because

- yarn is resolving prismjs ^1.14.0 and ^1.15.0 dependencies to 1.15.0 as opposed to 1.16.0 (which is the latest)
- prismjs 1.15.0 doesn't have `py` included as an alias for `python` while 1.16.0 does

**Comment by DSchau** [comment]
Published in gatsby-remark-prismjs@3.2.7

**Comment by ojeytonwilliams** [comment]
Thanks, @sidharthachatterjee and you're very welcome, @DSchau!  It's great you could sort this out so quickly.

**Pull request merged**